libass/ass.h

```diff
@@ -30,6 +30,21 @@
 extern "C" {
 #endif
 
+#if (defined(__GNUC__) && ((__GNUC__ > 3) || (__GNUC__ == 3 && __GNUC_MINOR__ >= 1))) || defined(__clang__)
+    #define ASS_DEPRECATE(msg)  __attribute__((deprecated(msg)))
```

Comment:
Nit: I'd prefer `ASS_DEPRECATED` here. Also, there's a doubled space here for no apparent reason.

Reply:
Done.